### ISSUE TITLE
Seed Phrase Created Wrong?

### ISSUE THREAD

**Issue by iamchrisryan** [Berry-Pool/nami-wallet#231]
Hey Gang,
   Created a new wallet, did what I should not have done and screenshotted my seed phrase. Wallet got hung up, another forum suggests reset. I reset and go to import existing wallet and seed phrase won't accept. Word # 24 is red outlined. Help?

**Comment by pascallapointe** [comment]
The words are checked against a collection of possible words. The position of the word doesn't matter.

Exemple, every word will pass the check in every position, but not every arrangement of the same groups of words can generate successfully a wallet key. You can try messing around with a seed, you'll notice that some arrangement will not let you "create" a wallet.

So, first step, try your word # 24 in any other field to check if it get outlined everywhere. If the words in a valid one, it should be accepted in any field under the "import" section. Doing so could maybe troubleshoot a weird browser issue or something.

Next, try importing that screenshot seed in another wallet. If the 24 seed is from a screenshot, and that you created a wallet from it, the seed will be importable in any other wallet client like Daedalus, CCVault, Yoroi, Adalite. If that operation is successful, that would indicate that something is wrong with your Nami installation or Browser

**Comment by iamchrisryan** [comment]
> The words are checked against a collection of possible words. The position of the word doesn't matter.

Yeah, the word in question is "propane" which is not one of the BIP39 words. I've tried most of the P's so far, attempting on Adalite because it seems the fastest interface on which to try.
